openlibrary/i18n/es/messages.po

```diff
@@ -1471,234 +1528,277 @@ msgstr ""
 "lectura. <a href=\"/help/faq/reading-log\">Más información</a> sobre el "
 "registro de lectura."
 
-#: account/readinglog_shelf_name.html:10
+#: account/reading_log.html
+msgid ""
+"There are no recent books in your feed. When you follow public accounts, "
+"their book updates will appear here."
+msgstr ""
+"No hay libros recientes en tu canal. Cuando sigas cuentas públicas, sus "
+"actualizaciones de libros aparecerán aquí."
+
+#: account/readinglog_shelf_name.html
 msgid "Want To Read"
 msgstr "Quiero leer"
 
-#: account/readinglog_stats.html:15 account/readinglog_stats.html:109
+#: account/readinglog_stats.html
 #, python-format
 msgid "My %(shelf_name)s Stats"
 msgstr "Mis estadísticas %(shelf_name)s"
 
-#: account/readinglog_stats.html:111
+#: account/readinglog_stats.html lib/nav_head.html
+msgid "My Books"
+msgstr "Mis libros"
+
+#: account/readinglog_stats.html
 #, python-format
 msgid ""
-"Displaying stats about <strong>%d</strong> books. Note all charts show only "
-"the top 20 bars. Note reading log stats are private."
+"Displaying stats about <strong>%(works_count)d</strong> books. Note all "
+"charts show only the top 20 bars. Note reading log stats are private."
 msgstr ""
-"Muestra estadísticas sobre <strong>%d</strong> libros. Ten en cuenta que "
-"todos los gráficos muestran solo las 20 primeras barras. Ten en cuenta que "
-"las estadísticas del registro de lectura son privadas."
+"Visualización de estadísticas sobre strong>%(works_count)d</strong> libros. "
```

Comment:
They should all be small stuff like this:

```suggestion
"Visualización de estadísticas sobre <strong>%(works_count)d</strong> libros. "
```

Reply:
I am not sure how I should proceed with the other issue (<strong> was my fault, sorry). In the message.pot file some html codes are also missing (</i>). Should we include them before in message.pot? 

I also found a strange string in message.pot that I didn't know if I should translate (line 1074). Shouldn't some code be used in "TITLE"?

> "Are you sure you want to leave the waiting list "
> "of<br/><strong>TITLE</strong>?"